### PR TITLE
update: Yew and jinko

### PR DESCRIPTION
**Pull request by Skallwar**

### CODE REVIEW

File: src/main.rs [hunk 1]
```diff
@@ -1,52 +1,59 @@
...
     type Message = Msg;
     type Properties = ();
 
-    fn create(_props: Self::Properties, link: ComponentLink<Self>) -> Self {
+    fn create(_ctx: &Context<Self>) -> Self {
         Self {
-            link,
             input: String::new(),
             output: Vec::new(),
+            jk_context: jinko::Context::new(Box::new(Reader {})),
```
Comment (CohenArthur) [inline src/main.rs]:
```suggestion
            jk_context: jinko::Context::new(Box::new(Reader)),
```

Comment (CohenArthur) [reply src/main.rs]: You should be able to use value structs like so

**Review by CohenArthur — APPROVED** [review]
Tiny nitpick :) LGTM!

File: src/main.rs (hunk 1 above)
Comment (Skallwar) [reply src/main.rs]: Fixed

**Pull request merged**